tensornetwork/network.py

```diff
@@ -20,19 +20,21 @@
 from typing import List, Optional, Union, Text, Tuple, Any
 import numpy as np
 import weakref
+from tensornetwork import config
 from tensornetwork import network_components
 from tensornetwork.backends import backend_factory
-
 Tensor = Any
 
 
 class TensorNetwork:
   """Implementation of a TensorNetwork."""
 
-  def __init__(self) -> None:
+  def __init__(self, backend: Optional[Text] = None) -> None:
     # TODO(chaseriley): Allow variable backend and default to global
```

Comment:
TODO is to be deleted since it is fixed.

Reply:
Done